pkgs/games/openttd/grfcodec.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgs }:
+
+stdenv.mkDerivation rec {
+  pname = "grfcodec";
+  version = "6.0.6";
+
+  src = fetchFromGitHub {
+    owner = "OpenTTD";
+    repo = "grfcodec";
+    rev = version;
+    sha256 = "15dl3v9n02j0zrlxazkv90zsp9z14swqz4c0q36b5hsfcdxjdhnl";
+  };
+
+  # prePatch = ''
+  #   sed -E -i 's,Image.VERSION,Image.PILLOW_VERSION,g' nml/version_info.py
+  # '';
+
+  buildInputs = with pkgs; [boost];
```

Comment:
`buildInputs = [ boost ]; ` should suffice (replace `pkgs` with `boost` on line 1)